src/cmd-buildextend-installer

```diff
@@ -206,17 +210,47 @@ def generate_iso():
         genisoargs += ['-r', '-l', '-sysid', 'PPC',
                        '-chrp-boot', '-graft-points']
     elif basearch == "s390x":
+        INITRD_ADDRESS = '0x02000000'
```

Comment:
Is there a link you could throw here to cross-reference where this comes from? Or does it just need to match whatever we use for the `.addrsize` file we create below?

Reply:
`https://github.com/weldr/lorax/blob/209bdd6911c782eca678d0efae3f2dd312727926/share/templates.d/99-generic/s390.tmpl#L6`

Reply:
This address is at offset of 32MB in memory where offset of 0 is for the kernel. s390x kernel devs chose this values many years ago predicting the kernel size would not exceed 32MB.

Reply:
Ahh perfect. Want to put that link as a comment there?

Reply:
Changed in latest commit.